examples/numbers/compact-notation.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Compact notation
+description: Print numbers in compact notation, such as 100K or 1.23M.
+related:
+  - examples/numbers/rounding-and-precision/
+---
+
+<mf2-interactive>
+
+```mf2
+{ 123456 :number notation=compact }
+```
+
+</mf2-interactive>
+
+<mf2-interactive>
+
+```mf2
+{ 123456 :number notation=compact compactDisplay=long }
+```
+
+</mf2-interactive>
+
+Compact notation is a way of expressing large numbers in a more concise form. It
+is commonly used in financial and scientific applications.
+
+The denominator of the compact form is a power of 10. In US English, examples
+are 1,000 (K), 1,000,000 (M), or 1,000,000,000 (B). In other locales, different
+denominators may be used, which may not always be a power of 1000. In India, for
+example, 1,00,000 (L) is used to represent 100,000.
```

Comment:
Use better line breaks. Semantic line breaks makes it easier to suggest edits, etc.

The examples (expecially the lakh example) is unclear. Perhaps:

```suggestion
For example, in US English (`en-US`) you might see:
- `100,000`  ⇒ `100K`
- `1,000,000` ⇒ `1M`
- `1,000,000,000` ⇒ `1B`

Other locales have different rules or might use different default
denominators. 
For example, English as used in India (`en-IN`) uses a different grouping strategy with units called "lakhs" and "crore":
- `1,00,000`  ⇒ `1L`
- `10,00,000` ⇒ `10L`
- `1,00,00,00,000` ⇒ `100Cr`
```